Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -275,6 +283,22 @@ inline HRESULT _RestoreD2DDrawingState(ID2D1DrawingStateBlock* drawingState) {
         _layerStack.emplace(baselineTarget.Get(), nullptr);
     }
 
+    inline void setFillColorSpace(CGColorSpaceRef colorspace) {
```

Comment:
please match the convention used in the rest of this class (capitalized method name)